### ISSUE TITLE
Mastodon link and icon issues

### ISSUE THREAD

**Issue by mdev88** [Jieiku/abridge#173]
Hello, 

I have noticed a couple of issues related to Mastodon:

1. When setting colorized icons ($ic) to `false`, the Mastodon icon is still colored.
2. When the Mastodon username is set, the link does not point to the correct URL, instead it adds the username to the local hostname. In my case the link points to `http://127.0.0.1:1111/@hackfun@rebel.ar`. Note that Mastodon usernames include both the username and the instance where that user exists. So in my example the resulting URL should look like this: `https://rebel.ar/@hackfun`

Thanks!

**Comment by Jieiku** [comment]
Try setting the full link for mastodon in config.toml:

```toml
mastodon = "https://mastodon.example.com/@username"
```

I believe that is what  your after, I will update the example in the distributed config.toml as well.

The issue with the colors relates to how I abused the color function used, it initially did not support multiple colors and I abused the function by misusing some of the parameters. I am currently rewriting the function so that it is more versatile.

I am having trouble with extracting an scss list from a map, it appears to be interpreting it as a string. Zola uses grass for its sass/scss but string.split is not yet implemented in grass but will be soon. I get the feeling that I should be able to extract the list from the map variable without it being interpreted as a string if I do it correctly, but not finding any clear examples of this yet.

Bottom line is that the color issue may take me a couple days, I will either find the answer/documentation or post a detailed question outlining the issue.

**Comment by mdev88** [comment]
No hurry, thank you for jumping to it so quickly. I am temporarily using colored icons so that the issue is not noticeable, and I'll change it back when it's fixed. 
 
The Mastodon link worked, I don't know why I didn't just try that, it was so obvious XD

Thanks!

**Comment by Jieiku** [comment]
I was able to resolve this (I also improved the contrast on the peertube icon when colorized)

https://github.com/Jieiku/abridge/commit/b2410f72fc3a8f81cc3bf3c1cfeca73b57c23e93
